healthcare/api-client/fhir/fhir_resources_test.py

```diff
@@ -102,40 +122,59 @@ def test_CRUD_search_resource(test_dataset, test_fhir_store, capsys):
         dataset_id,
         fhir_store_id,
         resource_type,
-        resource_id,
+        patient_resource_id,
     )
 
-    fhir_resources.conditional_update_resource(
+    fhir_resources.patch_resource(
         service_account_json,
         base_url,
         project_id,
         cloud_region,
         dataset_id,
         fhir_store_id,
         resource_type,
-        resource_id,
+        patient_resource_id,
     )
 
-    fhir_resources.patch_resource(
+    fhir_resources.delete_resource(
         service_account_json,
         base_url,
         project_id,
         cloud_region,
         dataset_id,
         fhir_store_id,
         resource_type,
-        resource_id,
+        patient_resource_id,
     )
 
-    fhir_resources.conditional_patch_resource(
+    out, _ = capsys.readouterr()
+
+    print(out)
+
+    # create_patient test
+    assert "Created Patient" in out
+    # get_resource test
+    assert "Got Patient resource" in out
+    # update_resource test
+    assert "Updated Patient resource" in out
+    # patch_resource test
+    assert "Patched Patient resource" in out
+    # delete_resource test
+    assert "Deleted Patient resource" in out
+
+
+def test_resource_versions(test_dataset, test_fhir_store, test_patient, capsys):
```

Comment:
I'd recommend splitting this up into multiple tests as well

Reply:
I would prefer to keep them in one because they all rely on each other (update has to happen before list which has to happen before get) rather than creating more fixtures. Does that sound OK?